packages/edit-site/src/components/global-styles/utils.js

```diff
@@ -10,3 +15,26 @@ export function getVariationClassNameFromPath( path ) {
 	}
 	return `is-style-${ path.split( '.' )[ 1 ] }`;
 }
+
+export function compareVariations( a, b ) {
+	if ( ! a.styles ) {
+		a.styles = {};
```

Comment:
This alters the original object `a`. It's generally a good idea to keep most utils pure. Do you think the same rule applies here?

Reply:
Excellent point, thank you, I will change this